website/blog/2024-06-24-run-ferretdb-cloudnativepg-kubernetes.md

```diff
@@ -0,0 +1,410 @@
+---
+slug: run-ferretdb-cloudnativepg-kubernetes
+title: 'Run FerretDB and Postgres Cluster using CloudNativePG on Kubernetes'
+authors: [alex]
+description: >
+  This guide will show you how to deploy FerretDB with a Postgres backend using CloudNativePG on Kubernetes.
+image: /img/blog/ferretdb-cloudnativepg.jpg
+tags: [compatible applications, tutorial, cloud, postgresql tools, open source]
+---
+
+![Run FerretDB and Postgres Cluster using CloudNativePG on Kubernetes](/img/blog/ferretdb-cloudnativepg.jpg)
+
+Managing, scaling, and deploying PostgreSQL database environments on Kubernetes has become increasingly popular and easier in recent times.
+A notable tool facilitating this process is CloudNativePG.
+
+<!--truncate-->
+
+[CloudNativePG](https://cloudnative-pg.io/) is an open source [PostgreSQL](https://www.postgresql.org/) operator developed by [EDB](https://www.enterprisedb.com/) under an [Apache 2](https://www.apache.org/licenses/LICENSE-2.0) license.
+It's designed for managing PostgreSQL workloads on [Kubernetes](https://kubernetes.io/) clusters on any private, public, hybrid, or multicloud environment.
+
+For users looking to run their MongoDB workloads on a truly open source software using [FerretDB](https://www.ferretdb.com/), CloudNativePG simplifies the deployment and management of production-level PostgreSQL backend on a Kubernetes cluster.
+
+In this guide, you'll learn how to deploy FerretDB and PostgreSQL on Kubernetes using CloudNativePG.
+
+## Prerequisites
+
+Ensure you have the following tools installed:
+
+- Kubernetes Cluster (You can set one up locally with [minikube](https://minikube.sigs.k8s.io/))
+- [`kubectl`](https://kubernetes.io/docs/reference/kubectl/)
+- [Helm](https://helm.sh/)
+
+Run the following command to install Helm:
```

Comment:
Two lines above you say I must have Helm installed :)
So, do I need helm installed, or will I install it as a part of this guide? :)

Reply:
We how how to install `helm` but not `kubectl` and others. What is special about `helm`?